ckan/lib/uploader.py

```diff
@@ -22,6 +22,21 @@
 _max_image_size = None
 
 
+def _copy_file(input_file, output_file, max_size):
+    input_file.seek(0)
+    current_size = 0
+    while True:
+        current_size = current_size + 1
+        # MB chunks
+        data = input_file.read(2**20)
```

Comment:
since you've moved this code, do you mind giving `2**20` a name? Just that and a rebase or merge to resolve the conflict and I'll merge, Thanks!